feat: Added readme (#66)
xmartlabs/fluttips: README.md

> <img src="readme/Fluttips_logo.png" width=“100%”/>

Reviewer: the image does not fit in the screen
![image](https://user-images.githubusercontent.com/8472881/213578443-bb4242ac-4aef-45f3-a94f-3fd9d5eed670.png)

# Fluttips

## Introduction

Fluttips is the first application that provides tips for programming in Flutter.

> It provides different sections, for example, you can see images with code explications and if you like, you can save the tip to see it later in favorites section.

Reviewer 2: ```suggestion
Some of its features include images and code explanations, making it easy to understand the tips provided.
Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.
```

> In a coming soon version, you will see videos of the widgets of the week, that provides you information about different components of Flutter.

Reviewer 2: ```suggestion
In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.
```

<table>
  <tr>
    <th>Images</th>
    <th><img src="readme/Images.gif" width="250"/></th>
  </tr>
  <tr>
    <th>Favorites</th>
    <th><img src="readme/Favorites.gif" width="250"/></th>
  </tr>
</table>

> The application was made with love by [Xmartlabs][xmartlabs] team and Fluttips is based on [Xmartlabs template][xmartlabs_template], which provides a clean architecture.   

Reviewer 2: ```suggestion
Made with ❤️ by [Xmartlabs][xmartlabs] and based on [our template][xmartlabs_template], which provides a clean architecture.
```

> The images tips are from [Vandadnp][Vandadnp]

Reviewer 2: ```suggestion
👉 The image tips are from [Vandadnp][Vandadnp]
```

and the videos are from [Widgets of the week][Widget_of_the_week].

## Contribute

- If you **want to contribute** please feel free to **submit pull requests**.
- If you **have a feature request** please **open an issue**.
- If you **found a bug** or **need help** please let us know. 

> 

Reviewer 2: ```suggestion
👉 If you enjoy using **Fluttips** we would love to hear about it! Drop us a line on [twitter](https://twitter.com/xmartlabs).
```

# Fluttips

## Introduction

Fluttips is the first app that brings together all the best tips and tricks for programming in Flutter in one place.
It allows you to access this information easily and quickly, making it a valuable resource for developers.
Some of its features include images and code explanations, making it easy to understand the tips provided.
Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.
In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.

<table>
  <tr>
    <th>Images</th>
    <th><img src="readme/Images.gif" width="250"/></th>
  </tr>
  <tr>
    <th>Favorites</th>
    <th><img src="readme/Favorites.gif" width="250"/></th>
  </tr>
</table>

Made with ❤️ by [Xmartlabs][xmartlabs] and based on [our template][xmartlabs_template], which provides a clean architecture.
👉 The image tips are from [Vandadnp][Vandadnp]

> 👉 The videos are from [Widgets of the week][Widget_of_the_week].

Reviewer: check the preview but new lines are missing

## Contribute

  👉 If you **want to contribute** please feel free to **submit pull requests**.

>   👉 If you **have a feature request** please **open an issue**.

Reviewer: link to new issue

  👉 If you **found a bug** or **need help** please let us know.

>   👉 If you enjoy using **Fluttips** we would love to hear about it! Drop us a line on [twitter](https://twitter.com/xmartlabs).

Reviewer: ref?

> <img src="readme/Fluttips_logo.png" width=“100%”/>

Reviewer: I'm not sure about the folder, what about /docs/assets?

[![Flutter CI](https://github.com/xmartlabs/fluttips/actions/workflows/flutter-ci.yml/badge.svg)](https://github.com/xmartlabs/fluttips/actions/workflows/flutter-ci.yml)

> ## Introduction

Reviewer: Do we need this title?

Fluttips is the first app that brings together all the best tips and tricks for programming in Flutter in one place.
It allows you to access this information easily and quickly, making it a valuable resource for developers.

Some of its features include images and code explanations, making it easy to understand the tips provided.
Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.

In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.

> <img src="readme/Images.gif" width=“500”/> <img src="readme/Favorites.gif" width="500"/>

Reviewer: the image does not look well 
![image](https://user-images.githubusercontent.com/8472881/213578589-565d6bf9-bfb5-475b-9d28-7193b21d04c8.png)

> <img src="readme/Images.gif" width=“500”/> <img src="readme/Favorites.gif" width="500"/>

Reviewer: images shouldn't say debug

Made with ❤️ by [Xmartlabs][xmartlabs] and based on [our template][xmartlabs_template], which provides a clean architecture.

📸 The image tips are from [Vandadnp][Vandadnp]

🎥 The videos are from [Widgets of the week][Widget_of_the_week].

## Contribute

  👉 If you **want to contribute** please feel free to **submit pull requests**.

  👉 If you **have a feature request** please **[open an issue][new_issue]**.

  👉 If you **found a bug** or **[need help][new_issue]** please let us know.

  👉 If you enjoy using **Fluttips** we would love to hear about it! Drop us a line on [twitter][twitter].

> 

Reviewer: license should be added here

> # Fluttips

Reviewer: We should add a banner with the logo

> 

Reviewer: include badges

## Introduction

Fluttips is the first app that brings together all the best tips and tricks for programming in Flutter in one place.
It allows you to access this information easily and quickly, making it a valuable resource for developers.

Some of its features include images and code explanations, making it easy to understand the tips provided.
Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.

In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.

<img src="readme/Images.gif" width="250"/>

> <img src="readme/Favorites.gif" width="250"/>

Reviewer: what about adding both in the same row?

Made with ❤️ by [Xmartlabs][xmartlabs] and based on [our template][xmartlabs_template], which provides a clean architecture.

👉 The image tips are from [Vandadnp][Vandadnp]

👉 The videos are from [Widgets of the week][Widget_of_the_week].

## Contribute

  👉 If you **want to contribute** please feel free to **submit pull requests**.

  👉 If you **have a feature request** please **[open an issue][new_issue]**.

>   👉 If you **found a bug** or **need help** please let us know.

Reviewer: how? maybe a link

Fluttips is the first app that brings together all the best tips and tricks for programming in Flutter in one place.
It allows you to access this information easily and quickly, making it a valuable resource for developers.

Some of its features include images and code explanations, making it easy to understand the tips provided.
Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.

In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.

<img src="readme/Images.gif" width="250"/>

> <img src="readme/Favorites.gif" width="250"/>

Reviewer: what about adding both in the same row?
